src/style/content/preferences/account.less

```diff
@@ -177,7 +177,7 @@
 .preferences-account-username-wrapper {
   position: relative;
   width: 256px;
-  min-height: @line-height-xl * 2;
+  min-height: calc(@line-height-xl * 2);
```

Comment:
why do you need a `calc` here? Doesn't seem to be a dynamic value 🤔

Reply:
need to use calc with the css variable is my guess

Reply:
But there is no css variable in there, only `less` variable (which are computed at compile time)

Reply:
ooohhhh the `less` variable contains a `css`  variable. 
Humm then we should probably use the css variable here, and avoid having the `less` proxy. WDYT?